pkgs/development/libraries/lief/default.nix

```diff
@@ -0,0 +1,14 @@
+{ stdenv, fetchzip }:
+
+fetchzip {
+  url = https://github.com/lief-project/LIEF/releases/download/0.9.0/LIEF-0.9.0-Linux.tar.gz;
+  sha256 = "1c47hwd00bp4mqd4p5b6xjfl89c3wwk9ccyc3a2gk658250g2la6";
+
+  meta = with stdenv.lib; {
+    description = "Library to Instrument Executable Formats";
+    homepage = https://lief.quarkslab.com/;
+    license = [ licenses.asl20 ];
+    platforms = platforms.linux;
+    maintainers = [ maintainers.lassulus ];
+  };
```

Comment:
`meta` belongs outside the `fetchzip`, not in it.

Reply:
This use is definitely okay. fetchzip output can be installed as a normal derivation so it is pretty much needed to use meta here. I also much prefer it to the all too common ```cp -r $src/* $out``` because it will be fixed output.

Reply:
Sorry, didn't know about that. Never seen this use before.